packages/compass-schema-validation/src/modules/is-loaded.js

```diff
@@ -0,0 +1,21 @@
+export const IS_LOADED_CHANGED = 'validation/namespace/IS_LOADED_CHANGED';
```

Comment:
I'm really not crazy about this boilerplate-heavy one whole file just for a boolean var pattern (especially when something like a selector could probably have figured it out based on validations state?), but it is what's used elsewhere as far as I can tell and I didn't want to introduce even more styles. At least not without discussion about which direction to go first.

Reply:
Definitely something we should chat about as a team. I think often we can use one reducer with more attributes in it to avoid all this boilerplate for each attribute having a reducer on the store. Some things in Compass we can also probably have live in state instead of on the store, so the store can be more useful for persisting data when unmounted or sharing data between components.

Reply:
I was thinking we'd probably only use stores/reducers for more global/shared state and the newer style react components that look like just render methods with all the hooks could help for a lot of this kind of internal state management. Hide as much as possible. But I'm no react expert (only used it a little bit and largely just read about it) and I think we should indeed have a discussion as a team before starting to change style like that.

There are probably tons of options to consider.